block-BHaaap/index.html

```diff
@@ -0,0 +1,78 @@
+<!---
+
+### Create a web page with the following characteristics:
+
+1. Heading element(h1) should be at the top of the page.
+2. After the heading, there should be a paragraph.
+3. After the paragraph add an image having all the corner rounded.
+4. Below the image, there should be a circle with red background.
+5. At the bottom of the page, there must be a footer containing few links.
+6. The H1 has 1px red solid borders, background color #D18C1D, and 10px of space between the content and the border (padding)
+7. The paragraph is contained in 600px by 400px boxes with 2px black dotted borders and background color #D1D631
+8. Links have a 3px blue dashed border on the bottom only and background color #A1A1A1
+
+- Using CSS resets is necessary.
+- Use semantic tags and keep the nesting and indentation proper.
+
+--->
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Box Model Assignment-1</title>
+</head>
+
+<body>
+    <header>
+        <h1>
+            Box Model-1
+        </h1>
+    </header>
+    <main>
+        <section>
+            <p>
+                Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                Aperiam numquam nihil officia,
+                dolores rerum natus voluptatem iusto vero qui iste nobis non atque ratione
+                reprehenderit labore tempora sint nisi sunt.<br><br>
+
+
+                Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                Aperiam numquam nihil officia,
+                dolores rerum natus voluptatem iusto vero qui iste nobis non atque ratione
+                reprehenderit labore tempora sint nisi sunt.<br><br>
+
+
+                Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                Aperiam numquam nihil officia,
+                dolores rerum natus voluptatem iusto vero qui iste nobis non atque ratione
+                reprehenderit labore tempora sint nisi sunt.<br><br>
+
+
+                Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                Aperiam numquam nihil officia,
+                dolores rerum natus voluptatem iusto vero qui iste nobis non atque ratione
+                reprehenderit labore tempora sint nisi sunt.
+            </p>
+
+            <img id="image" src="box-model.jpg" alt="Box Model">
+
+            <img id="circle" src="circle.png" alt="circle">
```

Comment:
This should be a circle build by you not an image.

Reply:
**Issue resolved**. *At that time was not aware that a circle can be drawn using a **div** tag*.